docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -110,23 +95,23 @@ A tax provider is ready to establish a connection with the Tax Provider API when
 
 For context, the [Tax Provider API Connection endpoints](/docs/rest-contracts/tax-app-connection) provide an added layer of security for tax providers. They set basic authentication credentials for the tax provider and these basic credentials are used to authenticate each API request to the tax provider from the associated store.
 
-If the tax provider supports all eligible stores, then they may choose to provide an account registration flow in their app iFrame in order to capture these basic authentication credentials from merchants. Learn more about designing the app UI [here](/docs/integrations/apps/guide/ui).
-
-If the tax provider is a private instance, then they may choose to provide the basic authentication credentials themselves.
+Offering an account login flow within the tax provider's app iFrame will commonly be required to capture these basic authentication credentials from merchants. An account registration flow may also be helpful for offering merchants the ability to create a new tax provider account. Learn more about designing the app UI [here](/docs/integrations/apps/guide/ui).
```

Comment:
```suggestion
An account login flow allows you to capture basic authentication credentials from merchants. It is best to provide a login flow within your app iFrame so that you can choose your own authentication method. You can also provide an account registration flow in your iFrame to help merchants create a new tax provider account. Learn more about [designing the app UI](/docs/integrations/apps/guide/ui).
```